src/main/java/com/gooddata/dataload/processes/Schedule.java

```diff
@@ -69,6 +70,27 @@ public Schedule(final DataloadProcess process, final String executable, final St
         this.links = Collections.emptyMap();
     }
 
+    /**
+     * Creates schedule, which is triggered by execution of different schedule
+     * @param process process to create schedule for
+     * @param executable executable to be scheduled for execution
+     * @param triggerSchedule schedule, which will trigger created schedule
+     */
+    public Schedule(final DataloadProcess process, final String executable, final Schedule triggerSchedule) {
```

Comment:
yes, we need a way to create schedule without having to specify cron expression. If both cron and triggerScheduleId is specified (= not null), you will not be able to create/update schedule.